### ISSUE TITLE
Hero Video: Pause button Focus

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#71]
The pause button does not have a visible focus indicator		

Include a similar focus indicator as the buttons.

**Comment by syarwick23** [comment]
Focus indicator is great! Can resolve this!
